packages/datetime/src/dateInput.tsx

```diff
@@ -90,6 +90,12 @@ export interface IDateInputProps extends IDatePickerBaseProps, IDateFormatProps,
      */
     fill?: boolean;
 
+    /**
+     * Whether the current day should be highlighted in the calendar.
+     * @default false
+     */
+    highlightCurrentDay?: boolean;
```

Comment:
since we're adding support to all day picker components, let's move this to `IDatePickerBaseProps`, so it gets inherited by all prop interfaces